1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,8 +16,9 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy =/* todo */ store
```

Comment:
Good use of method chaining. Please if you want to know more about it check: https://dev.to/isiakaabd/method-chaining-in-javascript-154a